src/components/BuildMenu/BuildMenuEntry/BuildMenuEntry.tsx

```diff
@@ -33,7 +33,7 @@ const BuildingMenuEntry = ({
       <button
         className="building-menu__entry"
         onClick={() => setModalOpen(true)}
-        disabled={!hasEnoughRessources(building.type)}
+       
```

Comment:
I think these buttons should be gray to directly indicate when the building can't be built yet. maybe conditionally add a classname (e.g. building-menu__entry--disabled)
Also, the empty line should be removed. Prettier should do this automatically if you set it up right: In VSCode, go to settings and search for formatter. Then set your settings like this:

<img width="994" alt="Bildschirmfoto 2024-05-25 um 11 28 53" src="https://github.com/Resaki1/taal/assets/35272402/28c60d79-a86e-46b8-8483-7bc18a605490">

This is one thing we should add to a github pipeline by the way